bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -260,6 +261,11 @@ data:
     if etcdctl $AUTH_OPTIONS endpoint health; then
         echo "Snapshotting the keyspace..." 1>&3 2>&4
         etcdctl $AUTH_OPTIONS snapshot save "/snapshots/db" 1>&3 2>&4
+        ln /snapshots/db /snapshots/db-$(date "+%Y-%m-%d_%H:%M") 1>&3 2>&4
```

Comment:
Please note that some systems may not allow filenames with colon.
Also ensure to use UTC time - it makes a lot of things easier.

Reply:
Alright, I added the `-u` flag for UTC time and replaced the colon with a dash :) Thanks!